### ISSUE TITLE
alt-d forces quit rizin

### ISSUE THREAD

**Issue by evilcel3ri** [rizinorg/rizin#4567]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Linux ee14787fa736 6.6.32-linuxkit #1 SMP PREEMPT_DYNAMIC Thu Jun 13 14:14:43 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
| File format of the file you reverse (mandatory)      | not applicable
| Architecture/bits of the file (mandatory)            | not application
| `rizin -v` full output, **not truncated** (mandatory)  | rizin 0.8.0 @ linux-x86-64 commit: c9fc24ecc4a86508352759cce367a069242b758c

### Expected behavior

`alt+d` should cut the character after the cursor and allow further input from the user.

### Actual behavior

After `atl+d` the next user input will exit rizin. 

### Steps to reproduce the behavior
- download the latest stable or dev version of rizin
- build if necessary, otherwise open rizin
- type anything
- enter `alt-d` 
- press space or any key
- rizin should exit

**Comment by wargio** [comment]
also fixed in dev.

**Comment by evilcel3ri** [comment]
oh ok, cool, on this commit? rizin 0.8.0 @ linux-x86-64 commit: https://github.com/rizinorg/rizin/commit/c9fc24ecc4a86508352759cce367a069242b758c

**Comment by wargio** [comment]
can't reproduce on latest dev. can you check the steps?

**Comment by evilcel3ri** [comment]
I rechecked the stuff, I'm wondering if that's coming from my keybaord. I've been testing that within docker and on macos and same issue both times for stable and dev.

**Comment by notxvilka** [comment]
@evilcel3ri does it still happen?

**Comment by evilcel3ri** [comment]
No, just tested on my ubuntu, and that issue doesn;t exist anymore :)
